examples/demo-prover/host/src/main.rs

```diff
@@ -0,0 +1,89 @@
+use demo_stf::app::create_demo_genesis_config;
+use demo_stf::app::{DefaultPrivateKey, NativeAppRunner};
+use demo_stf::config::from_toml_path;
+use demo_stf::config::Config as RunnerConfig;
+use jupiter::da_service::{CelestiaService, DaServiceConfig};
+use jupiter::types::NamespaceId;
+use jupiter::verifier::RollupParams;
+use methods::{ROLLUP_ELF, ROLLUP_ID};
+use risc0_adapter::host::Risc0Host;
+use serde::Deserialize;
+use sov_rollup_interface::services::da::DaService;
+use sov_rollup_interface::stf::{StateTransitionFunction, StateTransitionRunner};
+use sov_rollup_interface::zk::traits::ZkvmHost;
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct RollupConfig {
+    pub start_height: u64,
+    pub da: DaServiceConfig,
+    pub runner: RunnerConfig,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let rollup_config: RollupConfig = from_toml_path("rollup_config.toml")?;
+
+    let da_service = CelestiaService::new(
+        rollup_config.da.clone(),
+        RollupParams {
+            namespace: NamespaceId([115, 111, 118, 45, 116, 101, 115, 116]),
+        },
+    );
+
+    let sequencer_private_key = DefaultPrivateKey::generate();
+    let genesis_config = create_demo_genesis_config(
+        100000000,
+        sequencer_private_key.default_address(),
+        vec![
+            99, 101, 108, 101, 115, 116, 105, 97, 49, 122, 102, 118, 114, 114, 102, 97, 113, 57,
+            117, 100, 54, 103, 57, 116, 52, 107, 122, 109, 115, 108, 112, 102, 50, 52, 121, 115,
+            97, 120, 113, 102, 110, 122, 101, 101, 53, 119, 57,
+        ],
+        &sequencer_private_key,
+        &sequencer_private_key,
+    );
```

Comment:
nit: Can we update this to match the demo-rollup? That uses some constants to make the purpose of these magic values more clear

Reply:
Done!